corehq/db/tests/test_partition_config.py

```diff
@@ -0,0 +1,74 @@
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+from ..config import PartitionConfig
+from ..exceptions import PartitionValidationError
+
+TEST_PARTITION_CONFIG = {
+    'shards': {
+        'default': [0, 2],
```

Comment:
it might make sense to just have this be a `range`, but for the time being decided against it since it thought it would clutter the settings file even more